src/ui/inspector/ObjectName.vue

```diff
@@ -132,6 +139,8 @@ export default {
                     this.keyString = this.openmct.objects.makeKeyString(this.domainObject.identifier);
                     this.status = this.openmct.status.get(this.keyString);
                     this.statusUnsubscribe = this.openmct.status.observe(this.keyString, this.updateStatus);
+                } else if (selection[0][0].context.layoutItem) {
```

Comment:
Just a general question here, but I see `selection[0][0]` all over the codebase. What is this representing?

Reply:
It represents our gross selection API :)

More specifically, selection of an embedded object (eg. an object embedded in a layout) is represented by a path to the selected object (like `objectPath`, but the format is a little different because it can contain non domain-object things like draw elements in a layout). For simplicity's sake we decided to represent this with an array.

Then we added support for selecting multiple things at once, so our simple one dimensional array became a two dimensional array.

So, `selection[0][0]` is grabbing the thing the user actually clicked on to select (and not any of the object objects in the selected path), and is ignoring multi-select. Multi-select is only actually supported in display layouts (I think?), but the selection API always returns a two dimensional array anyway.

Reply:
We should improve the documentation around the selection API, and we should probably add some convenience functions for common operations (like the 90% case where you just want a single selected thing and you don't care about multi-select or the selection path)